### PR TITLE
4.1.7.3 Sending call dev - Failure: Peer User dropped call

### LINKED ISSUE #31: 4.1.7.3 Sending call request - failure: peer user dropped call
13.05.2021

### PR DESCRIPTION
**Pull request by filip6464**
Sending call - Failure: Peer User dropped call

### CODE REVIEW

File: UE/Application/States/ConnectedState.cpp [hunk 1]
```diff
@@ -113,5 +113,12 @@ void ConnectedState::handleUnknownRecipient()
     const auto index = gsl::narrow_cast<gsl::index>(numberOfMessages) - 1;
     context.smsDb.setMessageState(index, SmsState::Failed);
 }
+void ConnectedState::handleReceiveCallDrop()
+{
+    //todo #check
```
Comment (Blazej-Zielinski) [inline UE/Application/States/ConnectedState.cpp]: I think that we should inform user, that correspondent reject our call. Use `alertUser` function from `userPort`.

Example:
`context.user.alertUser("Call rejected");`

**Review by Blazej-Zielinski — CHANGES_REQUESTED** [review]
Inform user that call was dropped by correspondent using `alertUser` function

File: UE/Application/States/ConnectedState.cpp (hunk 1 above)
Comment (wiktorwieclaw) [inline UE/Application/States/ConnectedState.cpp]: ambiguous comment

File: UE/Tests/Application/ApplicationTestSuite.cpp [hunk 2]
```diff
@@ -257,6 +257,14 @@ TEST_F(ApplicationConnectedTestSuite, shallHandleReceiveCallAccepted)
     objectUnderTest.handleReceiveCallAccept(callingNumber);
 }
 
+TEST_F(ApplicationConnectedTestSuite, shallHandleReceiveCallDrop)
+{
+    //todo #check
```
Comment (wiktorwieclaw) [inline UE/Tests/Application/ApplicationTestSuite.cpp]: ambiguous comment

**Review by wiktorwieclaw — CHANGES_REQUESTED** [review]

**Review by wiktorwieclaw — APPROVED** [review]

**Pull request merged**